renderer.php

```diff
@@ -1973,7 +1976,7 @@ protected function simple_edit_template($t) {
 
                 $t->extrahtml
                 $hidden
-                <button class="rounded-pill btn btn-primary" type="submit">$t->submitlabel</button>
+                <button class="rounded-pill btn btn-primary submitreply" type="submit">$t->submitlabel</button>
```

Comment:
Was the submitreply class initially used for the JS solution? If so, can remove the class